docs/essentials/caching-dependencies-and-directories.md

```diff
@@ -13,10 +13,14 @@ them only when the dependency list changes.
 - Propagate a file from one block to the next.
 
 The cache is created on a per-project basis and available in every pipeline job.
-All cache keys are scoped per-project. The total cache size is 9.6GB.
+All cache keys are scoped per-project.
 
 The `cache` tool uses key-path pairs for managing cached archives. An archive
-can be a single file or a directory. Each archive automatically expires in 30 days.
+can be a single file or a directory.
+
+When using the [sftp storage backend](#sftp-backend), the total cache size is 9.6GB and each
```

Comment:
⚠️ most users of Semaphore read this page — how many will know what "sftp backend" is? I think we're leaking unnecessary complexity here...

Reply:
imo what people can understand and you should use as reference are plan names (Starter / Enterprise Cloud) and perhaps execution model hosted / self-hosted.